### DISCUSSION TITLE
Inflation - Adjusting Emission Factors to 2023 Spend Data

### DISCUSSION THREAD

**Discussion by pjvosmus** [USEPA/supply-chain-factors#68] [General]
BEA's Gross Industry Output Chained-Type Price Index for 2023 will not be published until September 2024, from what I understand.  I need to calculate 2023 emissions by early 2024. What is the most appropriate dataset to use to scale for inflation in this case?

**Comment by bl-young** [comment]
I am not aware of any sector specific inflation factors besides what BEA provides. (e.g., see #57). If you come up with other solutions please share as many people have this question. But a one or two year data lag for these types of data is not uncommon.

**Comment by pjvosmus** [comment]
We have historically used the Federal Reserve Economic Data (FRED) PPI Indexes by Industry. Not as granular as BEA, but better to use a broad brush than none at all?

**Reply by WesIngwersen** [reply]
We haven't evaluated using those or the BLS PPI indices but would be interested in the outcome of using those

**Comment by emehlhop** [comment]
When I pulled the data from BEA earlier this year, it showed Q1 for 2023. I used this value but can't seem to recreate that table now. I'm attaching the export I downloaded at the time.
[BEA Quarterly Inflation.xlsx](https://github.com/USEPA/supply-chain-factors/files/13396585/BEA.Quarterly.Inflation.xlsx)

**Reply by WesIngwersen** [reply]
The values are more aggregate than what we've used before, but yes more recent. Because BEA has done a comprehensive update of the industry and regional accounts I don't see any quarterly or detailed level gross output updates in their Data tool right now.
